src/test/java/com/sonyericsson/rebuild/RebuildValidatorTest.java

```diff
@@ -258,20 +267,20 @@ public void testWhenProjectWithParamsThenRebuildProjectExecutesRebuildOfLastBuil
                 new StringParameterDefinition("name", "defaultValue")));
 
         // Build (#1)
-        project.scheduleBuild2(0, new Cause.UserIdCause(),
-                new ParametersAction(new StringParameterValue("name", "test")))
+        final FreeStyleBuild freeStyleBuild = project.scheduleBuild2(0, new Cause.UserIdCause(),
+                        new ParametersAction(new StringParameterValue("name", "test")))
                 .get();
-        HtmlPage rebuildConfigPage = j.createWebClient().getPage(project,
-                "1/rebuild");
+        final HtmlPage buildPage = j.createWebClient().getPage(freeStyleBuild);
+        HtmlPage rebuildConfigPage = buildPage.getAnchorByText("Rebuild").click();
         // Rebuild (#2)
         j.submit(rebuildConfigPage.getFormByName("config"));
 
         HtmlPage projectPage = j.createWebClient().getPage(project);
         WebAssert.assertLinkPresentWithText(projectPage, "Rebuild Last");
 
         HtmlAnchor rebuildHref = projectPage.getAnchorByText("Rebuild Last");
-        assertEquals("Rebuild Last should point to the second build", "/jenkins/"
-                + project.getUrl() + "lastCompletedBuild/rebuild",
+        assertEquals("Rebuild Last should point to the second build",
+                "lastCompletedBuild/rebuild/parameterized",
```

Comment:
Test was actually okay before
```suggestion
        assertEquals("Rebuild Last should point to the second build",
                j.contextPath + "/" + project.getUrl() + "lastCompletedBuild/rebuild/parameterized",
```

Reply:
Thanks for the reminder.